.gitignore

```diff
@@ -28,4 +28,4 @@ doc/api/
 pubspec.lock
 
 # Node Modules
-tool/npm_template/node_modules/
+node/node_modules/
```

Comment:
Please update this path in `.gitattributes` as well, thanks!

Reply:
Done.